packages/rocketchat-statistics/server/functions/get.js

```diff
@@ -19,7 +19,7 @@ const wizardFields = [
 	'Allow_Marketing_Emails',
 ];
 
-RocketChat.statistics.get = function _getStatistics() {
+RocketChat.statistics.get = async function _getStatistics() {
```

Comment:
if you'll change this to `async` you'll also need to change where it is being called, as it will now return a promise.. see comment below

Reply:
I totally forgot about that.  🙈